packages/SwingSet/src/controller/controller.js

```diff
@@ -473,8 +473,8 @@ export async function buildVatController(
   };
   const initializationOptions = { verbose, kernelBundles };
   let bootstrapResult;
+  await null;
```

Comment:
Just curious: Why `null`?

No change suggested.

Reply:
I'm not sure of the lore behind this idiom, but I see it in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await among other places on the web.

Reply:
Unaware of those precedents. Good to know, thanks.